tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -89,6 +89,7 @@ def validate_dut_routes_exist(duthost, dut_dhcp_relay_data):
 
     for dhcp_server in dhcp_servers:
         rtInfo = duthost.get_ip_route_info(ipaddress.ip_address(dhcp_server))
+        # FIXME: this test looks iconnect, because we will receive a nexthop in case we have a default route
```

Comment:
incorrect.

Reply:
this is correct, the check is to prevent in some cases we do not receive the default route.

Reply:
should remove this FIXME comment.